### DISCUSSION TITLE
Managing Feed Data

### DISCUSSION THREAD

**Discussion by lowgrind** [bmoscon/cryptofeed#695] [Q&A]
Hey, I'm a python programmer more on the beginner end of the spectrum, but have worked with dataframes, NLP etc. I was wondering if anyone could help me on how i manage data from the feeds? I've got this basic code i've gotten from examples and i was wondering how i only print orders over 0.5 btc for example

I just haven't worked with live feeds like this so was unsure where to start on accessing the data like I do a REST API call (sorry for the noob question!)

Also, this project is honestly amazing, can't believe how much I am able to do just from looking at the examples. Thank you devs!

**Code**
```
from decimal import Decimal
from cryptofeed import FeedHandler
from cryptofeed.defines import TRADES
from cryptofeed.exchanges import Coinbase

async def trade(t, receipt_timestamp):
    assert isinstance(t.side, str)
    assert isinstance(t.amount, Decimal)
    assert isinstance(t.price, Decimal)
    assert isinstance(t.exchange, str)
    print(t)

def main():
    f = FeedHandler()
    f.add_feed(Coinbase(subscription={TRADES: ['BTC-USD']},callbacks={TRADES: trade}))
    f.run()

if __name__ == '__main__':
    main()
```

**Comment by lowgrind** [comment]
I ended up figuring this!
```
async def trade(t, receipt_timestamp):
    if t.amount > 0.5:
        assert isinstance(t.side, str)
        assert isinstance(t.amount, Decimal)
        assert isinstance(t.price, Decimal)
        assert isinstance(t.exchange, str)
        print(t)
```
Not sure if thats the efficient way to filter the data or not hah

**Reply by bmoscon** [reply]
yes, thats ideal. if you'd prefer to work with the data as a dictionary, you can call `to_dict()` on the `t` object.
